.github/workflows/main.yml

```diff
@@ -9,17 +9,13 @@ jobs:
       matrix:
         os:
           - ubuntu-20.04
-          - macos-11
```

Comment:
Was this intended to be replaced with `macos-latest` or is removal correct?

Reply:
I finally gave up on building with macos, due to weird dependency issues in the 3rd party packages, so in the end I just temporary dropped macos from the CI/CD pipelines. Though, with the latest I managed to get further that with the 11-th version, which didn't event start a build.